src/System.Data.Common/ref/project.json

```diff
@@ -1,14 +1,51 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.Threading.Tasks": "4.0.0",
-    "System.IO": "4.0.0"
+    "Microsoft.Win32.Primitives": "4.3.0-beta-24604-02",
```

Comment:
Ah, I thought this comment was on the src.  I expect the ref can be trimmed.  I'll do that.